src/pagingscrollview/src/NIPagingScrollView.h

```diff
@@ -75,6 +75,8 @@ typedef enum {
 // of the view.
 @property (nonatomic) CGFloat pageInset;
 @property (nonatomic) NIPagingScrollViewType type; // Default: NIPagingScrollViewHorizontal
+// Controls layout for RTL.
+@property (nonatomic) BOOL RTLEnabled;
```

Comment:
Would it be possible to use a flag to support versions of iOS prior to iOS 9 and then use the semanticContentAttribute for iOS 9+?
https://developer.apple.com/reference/uikit/uiview/1622461-semanticcontentattribute?language=objc